Jakefile

```diff
@@ -161,17 +161,30 @@ function generateChromeManifest() {
   });
 }
 
+/**
+ * Exports the files to a target, used to export into mozilla-central
+ *
+ */
+function exportFiles(target) {
+  let mcLocation = getMozillaCentralLocation();
+  let extTargetDir = path.join(mcLocation, target);
+  jake.rmRf(extTargetDir);
+  jake.cpR(BUILD_DIR, extTargetDir);
+
+  console.log(`Exported built sources into ${extTargetDir}`);
+}
+
 desc(`Builds the extension into the ${BUILD_DIR}/ directory`);
 task("build", ["building:cleanup", "building:copy"], () => {});
 
 desc("Exports the sources into mozilla-central");
 task("export-mc", ["build"], () => {
-  let mcLocation = getMozillaCentralLocation();
-  let extTargetDir = path.join(mcLocation, "browser/extensions/webcompat");
-  jake.rmRf(extTargetDir);
-  jake.cpR(BUILD_DIR, extTargetDir);
+  exportFiles("browser/extensions/webcompat");
+});
 
-  console.log(`Exported built sources into ${extTargetDir}`);
+desc("Exports the sources into the mozilla-central for android");
+task("export-mc-android", ["build"], () => {
```

Comment:
The "report site issue" thing on Fennec is not an extension, it [is a module natively built into Fennec](https://dxr.mozilla.org/mozilla-central/source/mobile/android/chrome/content/WebcompatReporter.js).

> First idea 2 platform shares 1 code, so we do only need 1 copy.

I would love to share the code between both builds, but system extensions for Desktop are in `browser/extensions` while system extensions for Fennec are in `mobile/android/extensions`. I had a quick look at the build chain, and there seems to be no link between those two builds in that regards. How would you share the sources?

> Second idea is like flyweb, they maintain 2 codes, then we use application info to export to folder.

I don't think I understand what you mean here. We export the exact same sources for both desktop and mobile, there is no actual code difference. The reason they can get exported individually is the fact that landing a system extension update in Desktop is much more complicated process-wise, and QA steps and thus landing timelines may differ. Could you elaborate what you mean here? :)